test/mocha/glob.js

```diff
@@ -21,4 +21,14 @@ describe("minify() with input file globs", function() {
         });
         assert.strictEqual(result.code, 'var print=console.log.bind(console);print("qux",function(n){return 3*n}(3),function(n){return n/2}(12)),function(n){print("Foo:",2*n)}(11);');
     });
+    it("should throw with non-matching glob string", function() {
+        assert.throws(function() {
+            Uglify.minify("test/input/issue-1242/blah.*");
+        }, "should throw file not found");
```

Comment:
It does return the original glob in `simple_glob` - the exception is raised due to `fs.readFileSync()` subsequently failing, i.e. being told to read a file that isn't there